dotnet-desktop-guide/net/winforms/whats-new/net60.md

```diff
@@ -0,0 +1,162 @@
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.MinimumSplashScreenDisplayTime?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.MinimumSplashScreenDisplayTime?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.Font?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.Font?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.HighDpiMode?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.ApplyApplicationDefaultsEventArgs.HighDpiMode?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.ApplyApplicationDefaults?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.HighDpiMode?displayProperty=fullName>
+  - <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.HighDpiMode?displayProperty=fullName>
+
+## Updated APIs
+
+- <xref:System.Windows.Forms.Control.Invoke?displayProperty=fullName> now accepts <xref:System.Action?displayProperty=fullName> and <xref:System.Func%601?displayProperty=fullName> as input parameters.
+- <xref:System.Windows.Forms.Control.BeingInvoke?displayProperty=fullName> now accepts <xref:System.Action?displayProperty=fullName> as an input parameter.
+- <xref:System.Windows.Forms.DialogResult?displayProperty=fullName> is extended with the following members:
+
+  - `TryAgain`
+  - `Continue`
+
+- <xref:System.Windows.Forms.Form?displayProperty=fullName> is extended with the following property:
+
+  - <xref:System.Windows.Forms.Form.MdiChildrenMinimizedAnchorBottom?displayProperty=fullName>
+
+- <xref:System.Windows.Forms.MessageBoxButtons?displayProperty=fullName> is extended with the following member:
+
+  - `CancelTryContinue`
+
+- <xref:System.Windows.Forms.MessageBoxDefaultButton?displayProperty=fullName> is extended with the following member:
+
+  - `Button4`
+
+- <xref:System.Windows.Forms.LinkClickedEventArgs?displayProperty=fullName> has now a new constructor and extended with the following properties:
+
+  - <xref:System.Windows.Forms.LinkClickedEventArgs.LinkLength?displayProperty=fullName>
+  - <xref:System.Windows.Forms.LinkClickedEventArgs.LinkStart?displayProperty=fullName>
+
+- <xref:System.Windows.Forms.NotifyIcon.Text?displayProperty=fullName> is now limited to 127 characters (from 63).
+
+## Enhanced features
+
+- High DPI improvements.
+TODO: @dreddy-work to provide text.
+- Windows 11 style default tooltip behavior makes the tooltip remain open when mouse hovers over it, and not disappear automatically. The tooltip can be dismissed by CONTROL or ESCAPE keys.
+- Microsoft UI Automation patterns work better with accessibility tools like Narrator and Jaws.
```

Comment:
@RussKie you might rephrase this to "assistive technology tools like Narrator and Jaws" - Assistive tech is sort of the industry standard.

Reply:
@merriemcgaw does it "work better" than the last time? Because we pointed this same thing out with 5.0

Reply:
Works betterer than ".NET 5.0 better" :D